### PR TITLE
[app management policies] update docs for adding restriction state

### PR DESCRIPTION
**Pull request by yogesh-randhawa**
**Instructions:** _Add any supporting information, such as a description of the PR changes, here._

---
> [!NOTE]
> The following guidance is for Microsoft employees only. Community contributors can ignore this message; our content team will manage the status.
<details><summary><i>After you've created your PR</i>, expand this section for tips and additional instructions.</summary>

- **do not merge** is the default PR status and is automatically added to all open PRs that don't have the **ready to merge** label.
- Add the **ready for content review** label to start a review. Your PR won't be reviewed until you add this label.
- If your content reviewer requests changes, review the feedback and address accordingly as soon as possible to keep your pull request moving forward. After you address the feedback, remove the **changes requested** label, add the **review feedback addressed** label, and select the **Re-request review** icon next to the content reviewer's alias. If you can't add labels, add a comment with `#feedback-addressed` to the pull request.
- After the content review is complete, your reviewer will add the **content review complete** label. When the updates in this PR are ready for external customers to use, replace the **do not merge** label with **ready to merge** and the PR will be merged within 24 working hours.
- Pull requests that are inactive for more than 6 weeks will be automatically closed. Before that, you receive reminders at 2 weeks, 4 weeks, and 6 weeks. If you still need the PR, you can reopen or recreate the request.

For more information, see the [Content review process summary](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/614263/Content-workflow).

</details>

### CODE REVIEW

File: api-reference/beta/resources/keycredentialconfiguration.md [hunk 1]
```diff
@@ -17,12 +17,13 @@ Represents a key credential configuration object that contains properties to con
...
+| Property                                    | Type                            | Description |
+| :------------------------------------------ | :------------------------------ | :--------------------------------------------------------------------------------------------------------------------------------------- |
+| restrictionType                             | appKeyCredentialRestrictionType | The type of restriction being applied. Possible values are `asymmetricKeyLifetime`. Each value of restrictionType can be used only once per policy.|
+| state                                       | appManagementRestrictionState   | String value indicating the how the restriction is evaluated. The possible values are: `enabled`, `disabled`. If `enabled`, the restriction will be evaluated. If `disabled`, the restriction will not be evaluated.|
```
Comment (aricrowe57) [inline api-reference/beta/resources/keycredentialconfiguration.md:23]: Wording 'will be evaluated' is slightly confusing to me.  I suggest:

```suggestion
| state                                       | appManagementRestrictionState   | String value indicating if the restriction will be enforced. The possible values are: `enabled`, `disabled`. If `enabled`, the restriction will be evaluated and enforced. If `disabled`, the restriction will not be enforced, and no properties on the restriction will be evaluated. |
```

File: api-reference/beta/resources/keycredentialconfiguration.md [hunk 2]
```diff
@@ -17,12 +17,13 @@ Represents a key credential configuration object that contains properties to con
...
+| restrictForAppsCreatedAfterDateTime         | DateTimeOffset                  | Datetime timestamp indicating the date on or after which the apps are created that the policy restriction should apply to. For existing applications, the enforcement date can be back dated.|
+| maxLifetime                                 | Duration                        | String value indicating as maximum lifetime for key expiration, defined as an ISO 8601 duration. For example, `P4DT12H30M5S` represents a duration of four days, twelve hours, thirty minutes, and five seconds. This property is required when **restrictionType** is set to `keyLifetime`.|
+| certificateBasedApplicationConfigurationIds | Collection (Guid)               | Collection of Guids that point to the [certificateBasedApplicationConfiguration](../resources/certificatebasedapplicationconfiguration.md) that contains the collection of allowed root and intermediate certificate authorities.|
```
Comment (madansr7) [inline api-reference/beta/resources/keycredentialconfiguration.md:26]: name ordering should be alphabetical.

File: api-reference/beta/resources/enums.md [hunk 3]
```diff
@@ -587,6 +587,14 @@ Namespace: microsoft.graph
 | selected           | 2     | The policy applies to specific users or groups in the organization.            |
 | unknownFutureValue | 3     | Evolvable enumeration sentinel value. Don't use.                              |
 
+### appManagementRestrictionState values 
+
+|Member|
+|:---|
+|enabled|
```
Comment (madansr7) [inline api-reference/beta/resources/enums.md:594]: Shouldn't 'disabled' be the first and the default state of restriction?

**Comment by yogesh-randhawa** [comment]
Created new PR on the private repo https://github.com/microsoftgraph/microsoft-graph-docs/pull/24588

**Pull request closed without merging**